include/utilities/CSV_Reader.h

```diff
@@ -30,6 +30,13 @@ inline std::vector<std::vector<std::string> > CSVReader::getData()
 {
 	std::ifstream file(fileName);
 
+        if(file.fail()){
+            /// \todo TODO: Return appropriate error
+            throw std::runtime_error("Error: Forcing file " + fileName + " does not exist.");
+
+            /// \todo Potentially only output warning and fill array with sentinel values.
```

Comment:
So we do want to go with sentinel values instead of just killing the program here?

Reply:
I think this is an appropriate fatal error